config/cmake-presets/hidden-presets.json

```diff
@@ -299,7 +303,8 @@
       "inherits": "ci-base",
       "generators": [
         "ZIP"
-      ]
+      ],
+      "configurations": ["Release"]
```

Comment:
This one is "Configurations", while the others are "Configuration".  Is the difference correct?